build-packages/webpack-i18n-runtime/src/util/__.js

```diff
@@ -11,7 +11,7 @@ function translateString(string) {
 }
 
 function getTranslatedStringWithInjectedValues(string, values) {
-    return injectValues(translateString(string), values);
+    return injectValues(translateString(string), ...values);
```

Comment:
Unclear what does this fix do. Please add comment or make sure this is left here intentionally. This seems to have nothing to do with HTML plugins.

Reply:
Referencing [pull request](https://github.com/scandipwa/create-scandipwa-app/pull/12) with all issues, attached within the conversation

Reply:
Referencing [pull request](https://github.com/scandipwa/create-scandipwa-app/pull/12) with all issues, attached within the conversation

Reply:
Probably a rebase issue. This indeed is related to the thing mentioned by @winniepukki